monai/metrics/metric.py

```diff
@@ -37,6 +37,12 @@ def __call__(self, *args: Any, **kwargs: Any) -> Any:
         """
         raise NotImplementedError(f"Subclass {self.__class__.__name__} must implement this method.")
 
+    def __repr__(self):
+        return self.__class__.__name__
+
+    def __str__(self):
```

Comment:
Here `__str__` is producing something readable and concise which is fine, in other contexts it might make sense to include some state information if this wasn't too lengthy.

Reply:
I've removed `__repr__`, just having the `__str__` will reduce a bit of code for me either way!